lms/templates/base.html.jinja2

```diff
@@ -32,7 +32,19 @@
       {% block content %}{% endblock %}
     </div>
 
-    <script type="application/json" class="js-config">{{ js_config|tojson }}</script>
-    {% block scripts %}{% endblock %}
+    {% block scripts %}
+      <script type="application/json" class="js-config">{{ js_config|tojson }}</script>
```

Comment:
I couldn't find any other references to `js-config` in the codebase so I think this is unused. The LMS frontend JS is using `js-hypothesis-config`, which is confusing and conflicts with the config used for the client config.

Reply:
Yeap ... That's going to be fixed but is unrelated to what's in this PR.